aiosmtpd/docs/handlers.rst

```diff
@@ -183,7 +183,7 @@ those methods of the handler instance will override the built-in methods.
 
   :boldital:`server` is the instance of the ``SMTP`` class invoking the AUTH hook.
   This allows the AUTH hook implementation to invoke facilities such as the
-  ``push()`` and ``_auth_interact()`` methods.
+  ``push()`` and ``challenge_auth()`` methods.
```

Comment:
Deprecating `_auth_interact()` because it's 'private'. Couldn't start with `auth_` though, so used a new name.